python/salpytools/salpylib.py

```diff
@@ -107,25 +106,24 @@ def __init__(self, Device='atHeaderService', default_state='OFFLINE',
             self.load_state_enumeration()
         except Exception:
             LOGGER.warning("Cannot load state enumeration -- will use library instead")
-            self.detailedState_enum = states.state_enumeration
+            self.State_enum = states.state_enumeration
 
     def load_state_enumeration(self):
         """
-        Load up into dictionaries the SummaryState and DetailedState
+        Load up into dictionaries the State
         enumeration by getting the attributes of the shape
-        'atHeaderService_shared_SummaryState_DisabledState' and
-        'atHeaderService_shared_DetailedState_DisabledState' for example
+        'SALPY_ATCamera.SAL__STATE_DISABLED'
```

Comment:
Shouldn't that be ATHeaderService or is it just a generic holder to point at the state constants?

Reply:
Just a generic holder